uk.co.compucorp.civicrm.civihrcore/CRM/CiviHRCore.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+class CRM_CiviHRCore {
```

Comment:
maybe **CRM_CiviHRCore_Core** ... isn't this is the common pattern we use in other extensions